proarc-common/src/main/java/cz/cas/lib/proarc/common/export/mets/Const.java

```diff
@@ -102,14 +100,14 @@
     public static final String UC_GRP_ID = "UC_IMGGRP";
     public static final String TXT_GRP_ID = "TXTGRP";
 
-    public static final List<String> PSPElements = new ArrayList<String>();
+    public static final Collection<String> PSPElements = new ArrayList<>();
```

Comment:
Chtělo by to rozhodnout jestli půjdeme cestou co největší abstrakce vůči požadované funkcionalitě (Collection, List, Set), anebo jen "klasicky" rozhraním daného typu kolekce (List, Set) a pak to sjednotit.

https://github.com/proarc/proarc/blob/c8c5a87efcd96df7d29b55fae6739e5162202d96/proarc-common/src/main/java/cz/cas/lib/proarc/common/export/mets/Const.java#L41-L43